src/Kerbalism/UI/FileManager.cs

```diff
@@ -131,12 +132,12 @@ static void Render_file(Panel p, uint partId, File file, Drive drive, bool short
 			if (file.transmitRate > 0.0)
 			{
 				if (file.size > 0.0)
-					exp_tooltip = Lib.Color(Lib.BuildString(exp_tooltip, "\nTransmitting at ", Lib.HumanReadableDataRate(file.transmitRate), " : <i>", Lib.HumanReadableCountdown(file.size / file.transmitRate), "</i>"), Lib.Kolor.Cyan);
+					exp_tooltip = Lib.Color(Lib.BuildString(exp_tooltip, "\n"+Localizer.Format("#KERBALISM_FILEMANAGER_TransmittingRate", Lib.HumanReadableDataRate(file.transmitRate)), " : <i>", Lib.HumanReadableCountdown(file.size / file.transmitRate), "</i>"), Lib.Kolor.Cyan);//Transmitting at <<1>>
```

Comment:
don't use `+` and don't use the `<<1>>`, change to : 
`exp_tooltip = Lib.Color(Lib.BuildString(exp_tooltip, "\n", Localizer.Format("#KERBALISM_FILEMANAGER_TransmittingRate"), " ", Lib.HumanReadableDataRate(file.transmitRate), " : <i>", Lib.HumanReadableCountdown(file.size / file.transmitRate), "</i>"), Lib.Kolor.Cyan);`
with the string being `Transmitting at`

(same remark in other parts in this file)

Reply:
Same here, it's a sentence:
Compare
```
#KERBALISM_FILEMANAGER_TransmittingRate = Transmitting at <<1>>
```
and
```
#KERBALISM_FILEMANAGER_TransmittingRate = 以<<1>>速率传输
```

You can probably escape string interpolation by turning this into a  `label: value` thing as you suggest, but you would have to change the string to a noun phrase, `Transmission rate:` or something along those lines.

Reply:
Yup, well keep the localizer call, but at least change the `+` to a `,`

Reply:
Changed